pages/super_search_page.py

```diff
@@ -130,10 +129,14 @@ def results_facet(self):
     def is_column_in_list(self, column_name):
         return column_name in [column.column_name for column in self.columns]
 
-    class SearchResultHeader(Page):
+    class SearchResultHeader(object):
 
         _table_header_name_locator = (By.CSS_SELECTOR, '#reports-list thead th')
 
+        def __init__(self, selenium, timeout):
+            self.selenium = selenium
+            self.timeout = timeout
```

Comment:
Page regions should accept `base_url` and `selenium` arguments. The `timeout` argument is reasonable if we need to modify it, but by default it should be handled by parent class. If you want to, you could introduce a `PageRegion` class in `page.py` to take care of this, which is what we've done in other suites.